pkg/runner/deployer/ssh.go

```diff
@@ -135,3 +147,54 @@ func (p *SSH) Run(cmdSegs []string, handler DeployerHandler) (err error) {
 
 	return nil
 }
+
+type FSStats struct {
+	Path       string `json:"path"`
+	BlockSize  uint64 `json:"block-size"`
+	FreeBlocks uint64 `json:"free-blocks"`
+}
+
+func (fss *FSStats) FreeBytes() uint64 {
+	return fss.BlockSize * fss.FreeBlocks
+}
+
+func (p *SSH) FSStats(path string) (stats FSStats, err error) {
```

Comment:
I don't think this needs to be on the SSH struct.  Just make FSStats take an SSH client pointer as its input,  and then it can use it.